storage_service/storage_service/settings/base.py

```diff
@@ -485,3 +485,13 @@ def is_true(env_str):
 # - locations.models.pipeline
 # - locations.models.dspace
 INSECURE_SKIP_VERIFY = is_true(environ.get("SS_INSECURE_SKIP_VERIFY", ""))
+
+PROMETHEUS_ENABLED = is_true(environ.get("SS_PROMETHEUS_ENABLED", ""))
```

Comment:
Should we add the new `SS_PROMETHEUS_ENABLED` env variable to the [Application-specific environment variables section of the README](https://github.com/artefactual/archivematica-storage-service/blob/83d7b5a7da79c158cb99bd0e3426b92fdde0d3f0/install/README.md#application-specific-environment-variables)?

Reply:
Good catch! Updated.